scripts/SMD_chip_package_rlc-etc/SMD_package_rlc-etc.py

```diff
@@ -334,10 +344,23 @@ def generateFootprint(self, device_size_data, footprint_group_data):
                         help='list of files holding information about what devices should be created.')
     parser.add_argument('--global_config', type=str, nargs='?', help='the config file defining how the footprint will look like. (KLC)', default='../tools/global_config_files/config_KLCv3.0.yaml')
     parser.add_argument('--series_config', type=str, nargs='?', help='the config file defining series parameters.', default='config_KLCv3.0.yaml')
-    parser.add_argument('--ipc_definition', type=str, nargs='?', help='the ipc definition file', default='ipc7351B_smd_two_terminal_chip.yaml')
+    parser.add_argument('--ipc_definition', type=str, nargs='?', help='the ipc definition file', default='ipc7351B_smd_two_terminal.yaml')
+    parser.add_argument('--ipc_density', type=str, nargs='?', help='IPC density level (L,N,M)')
     parser.add_argument('--force_rectangle_pads', action='store_true', help='Force the generation of rectangle pads instead of rounded rectangle (KiCad 4.x compatibility.)')
     args = parser.parse_args()
 
+    # if the user requests an IPC density, put that and footprint suffix in a list
+    # otherwise, use nominal density with no suffix
+    if args.ipc_density is None or args.ipc_density not in ['l', 'L', 'n', 'N', 'm', 'M']:
```

Comment:
```suggestion
    if args.ipc_density is None or args.ipc_density.lower() not in ['l', 'n', 'm']:
```

Reply:
I think there's an even better way to do this. I believed this could be improved but my brain was feeling worn out as I mentioned above; hopefully now inspiration has struck.